randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -18,7 +18,9 @@ Extra - asset_id: collision_camera_034
   * Extra - actor_type: weightactivatedplatform
   * Extra - start_point_actor_name: ST_FromArea09
   > Before Diggernaut Area
-      Morph Ball and After Area 6 - Transport to Area 7 Grapple Block Top
+      All of the following:
+          Morph Ball and After Area 6 - Transport to Area 7 Grapple Block Top
+          Plasma Beam or Defeat Non-Counterable Enemies
```

Comment:
No idea how this here actually works: The Grapple Block is already pulled by default in the starter settings iirc.  Which means you don't need to collect the event? Is this taken into account somewhere?

Also: Can you bypass `Defeat Non-Counterable Enemies` with damage boost? (This would apply to a lot of changes here)

Reply:
Yes the event is handled in the bootstrap iirc.

And yeah there's definitely room for improvement here as tricks could be added, but I wanted to at least make trickless somewhat better and more consistent.  It will have to be revisited later on